src/state/research.js

```diff
@@ -33,6 +33,20 @@ const research = merge(cloneDeep(jobBase), cloneDeep(jobSingleAction), {
 		jobId() {
 			return "research";
 		},
+		hasBountyItems(state, getters, rootState, rootGetters) {
+			for (let [itemId, requiredCount] of Object.entries(state.researchBountyItems)) {
+				let count = rootGetters["inventory/bank"][itemId];
+				count = count ? count : 0;
+				/* code to count worn items
```

Comment:
not sure i want to bother with this since it's buggy anyways (negative item bugs)
might not even put equippables into the destructive analyzer pool anyways, not sure.
problem is that the component im using to list items probably counts equipped items, and it would be weird if it appeared you had enough but actually didnt because youre wearing one.

Reply:
My biggest regret was allowing the "use equipped items in actions" PR go through. There's infinite bugs from that, so I can't really fault you for not trying it here.

Reply:
Yep. I think in testing this is actually slightly better bug-wise. If you allow equipped items, it will cause some weird negative item count bugs, but if you don't allow it, the worst that happens is it appears as though you have enough but won't let you actually deposit the item until you unequip it. Could probably be handled better by checking the required items against the worn items and just flatly rejecting the player with a toast message that tells them to unequip it first